### PR TITLE
Map-directions

### PR DESCRIPTION
**Pull request by itsLucas-h**
Testers - Josh
Josh has tested and confirmed all functionalities works as intended. 

Feel free to look at code and comment on anything. 

As a user, I want the map to provide directions to a restaurant I choose, so that I can navigate how to get there without using a different app.

### CODE REVIEW

File: components/CustomCallout.tsx
Comment (Joshua-Frost) [inline components/CustomCallout.tsx:1]: Nice work on the custom one, looks tidier

**Review by Joshua-Frost — APPROVED** [review]
Nice one, looks good to me

**Review by FranTheCaveman — APPROVED** [review]
## Looks great! 👍

**Review by FranTheCaveman — COMMENTED** [review]
One small thing, the navigation path remains when filtering out the eating spot marker with the categories/search filter

**Pull request merged**